### DISCUSSION TITLE
Query results Export File Name

### DISCUSSION THREAD

**Discussion by MWLambRN** [dbeaver/dbeaver#19301] [Q&A]
Apologies if this is a silly question, but I'm new to DBeaver and SQL queries in general. 
I've created several SQL queries and renamed and 'saved as' to change the query name from the default format "<db>Script NN" to something more meaningful - like "<db>MyTestQuery".  After I execute the query, I export the data (via the 'Export data' option at the bottom of the results window). I go through the Output steps, and have as part of my settings:
"${timestamp}_${scriptFilename}"
The output file has the {timestamp} component, but it retains the old/default script file name. 
I've attempted Save, Save As, Rename with the query (.sql file) - but the file name always retains the default. 
I looked through the support documentation, and this behavior doesn't seem to fit what I'm seeing there, and I can't find any additional information...so I'm here, asking silly questions that probably have a super-easy answer and I'm just missing something.

**Comment by E1izabeth** [comment]
Seems that it's a bug. Please, create an issue.
